docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,726 @@
+- (again) To exit, type ":q!" in normal mode
+- The "q!" means quit without saving, so you can also do ":wq" to write and then quit
+- Try out the `vimtutor` command if you want to learn how to actually use vim.
+
+#### nano
+
+`nano` is another terminal-based text editor. It's easier to use compared to vim and it has the 
+controls on the bottom of the screen, but it's (in Conner's opinion) a horrible experience 
+that leaves him annoyed every time he uses it. If you're dedicated to learning a really fast
+text editor, learn vim, but I'm leaving this here for people who don't want to.
+
+Example:
+```
+nano <filepath>
+```
+
+#### sudo
+
+`sudo` stand for "super-user do," and it's required at times where elevated permissions are required.
+This may be when you're dealing with system configurations or programs that need access to possibly 
+sensitive information, so your password will be required to allow it to make changes. This is sometimes a good
+thing, but you should definitely not just start using it whenever a program isn't working in a way
+you want it to. Again, remember that this gives permission to whatever program you're calling it with,
+so you should be a little careful about using it. You can read a little bit more about that 
+[here](https://www.linux.com/news/sudo-or-not-sudo-question/).
+
+Format:
+```
+sudo <some other command>
+```
+
+- An example of a time where it may be useful to use it is when globally installing an npm package. 
+npm will need access to special folders, and so sudo will likely need to be used.
+
+Example:
+```
+# Globally install the create-react-app package using npm
+sudo npm install -g create-react-app
+```
+
+#### man
+
+`man` shows the manual for a command. A lot of times, it can be the same as just calling 
+a command with the `--help` or `-h` flags, but it will often provide more information.
+It will usually show other flags, how the command should be formatted, and general
+options for its use.
+
+Format:
+```
+man <command>
+```
+
+- When you use `man`, it will open up a program called `less`, which is very similar to `vim`.
+The difference between `vim` and `less` is that `vim` is an editor, while `less` is more of
+a viewer. Therefore, `less` has much fewer commands. The only commands you need to know for less
+are the up/down arrow keys (or "j"/"k" if you're used to `vim`) and "q" to quit 
+(notice how there's no ":" preceding or "!" following it).
+- It seems like `man` may not come with Git bash, so just stick with `--help` and `-h` for now.
+    - The `--help` and `-h` flags are the same, but sometimes some programs will only implement
+    one or the other, so it's useful to know both.
+
+Example:
+```
+# Show the manual for ls
+man ls 
+
+# Use the --help flag
+ls --help
+
+# Use the -h flag
+ls -h
+```
+
+### Build Tools
+
+You'll probably have seen the following terms used quite a bit, so we'll go over the basics of
+what they do and how to use them.
+
```

Comment:
Add a sentence that these commands only work in the directories with your projects. Also mention that npm is for the frontend repo and maven is for the backend repo

Reply:
did it.